### ISSUE TITLE
a new config that make Xwayland applications scaling by Xft.dpi

### ISSUE THREAD

**Issue by Dawn-Xu-helloworld** [swaywm/sway#7970]
- **Description:**
Add a feature that allows applications running under Xwayland to not be scaled directly, but instead to follow the DPI settings specified in ~/.Xresources, just like scaling in KDE

**Comment by weofiw** [comment]
This would be really nice for running steam games at full 4k resolution instead of the scaled down resolution. Right now I'm manually switching scale back to 1.0 before running games and switching back later.

**Comment by runxiyu** [comment]
Agreed, this would be very useful. I'll assume (without much research) that there are a significant group of users for whom games are the majority of Xwayland programs that they use.
